src/org/javarosa/core/model/Constants.java

```diff
@@ -81,11 +90,9 @@
     public static final int CONTROL_AUDIO_CAPTURE = 12;
     public static final int CONTROL_VIDEO_CAPTURE = 13;
     public static final int CONTROL_OSM_CAPTURE = 14;
-    public static final int CONTROL_FILE_CAPTURE = 15; // generic upload
+    /** generic upload */
+    public static final int CONTROL_FILE_CAPTURE = 15;
 
-    /** constants for xform tags */
+    /* constants for xform tags */
```

Comment:
Same question here.

Reply:
No, this is sort of a description of a group that follows, and has no way to express it in a doc comment.

Reply:
Ahh, makes sense.

Reply:
Sorry for stepping in, but we're trying to improve the support for automatic reformatting on Briefcase and looking into extending that to other projects as well.

How do these block comments react to automatic reformatting? 

If they're just one-line comments, wouldn't it be better to use one-line comment syntax `// blablabla`, which won't be relocated/reformatted by the automatic reformatting tool?

Reply:
Hi @ggalmazor. They are doc comments (JavaDoc). They allow IDEA users to press F1 (depending on key bindings) anywhere the fields are used to get the description (see the picture). This change was simply to enable this. 

I just changed the verb in the title in the description from “fix” to “improve”, to be consistent with this being an incremental change.

I agree with you about better names, and valueless redundant comments. Going further, most of these should be changed to Enums.

Feel free to continue the work. I was just solving something that was bothering me at the moment (while looking at something in Collect).

Reply:
Thanks, @dcbriccetti. I just wanted to discuss it a little bit to see if this could get traction.